tools/devservices_healthcheck.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import annotations
+
+import os
+import subprocess
+from time import sleep
+
+
+def run_cmd(
+    args: list[str],
+    *,
+    retries: int = 3,
+    timeout: int = 5,
+) -> int:
+    for retry in range(1, retries + 1):
+        try:
+            returncode = subprocess.call(args)
+        except OSError:
+            raise SystemExit(1)
+
+        if returncode != 0:
+            sleep(timeout)
+        else:
+            return 0
+
+    raise SystemExit(1)
+
+
+def main() -> None:
+    # Available health checks
+    postgres_healthcheck = ["docker", "exec", "sentry_postgres", "pg_isready", "-U", "postgres"]
+    kafka_healthcheck = [
+        "docker",
+        "exec",
+        "sentry_kafka",
+        "kafka-topics",
+        "--zookeeper",
+        "127.0.0.1:2181",
+        "--list",
+    ]
+
+    healthchecks = list()
+    healthchecks.append(postgres_healthcheck)
+    if os.getenv("NEED_KAFKA") == "true":
+        healthchecks.append(kafka_healthcheck)
```

Comment:
```suggestion
    healthchecks = [postgres_healthcheck]
    if os.getenv("NEED_KAFKA") == "true":
        healthchecks.append(kafka_healthcheck)
```

sorry I didn't catch this one earlier